tests/requirements.txt

```
@@ -1,7 +1,7 @@
-anaconda-client
 beautifulsoup4
 chardet
 conda >=22.11.0
+conda-forge::anaconda-client
```

Revert since `anaconda-client` is now available on `main`:

```suggestion
anaconda-client
beautifulsoup4
chardet
conda >=22.11.0
```